test/stream/api/source/version-detail-ok/004-patch-settings/request.json

```diff
@@ -0,0 +1,21 @@
+{
+  "path": "/v1/branches/default/sources/my-source/settings",
```

Comment:
Why is patch settings called twice in the e2e tests? (`003-patch-settings` and `004-patch-settings`)

Reply:
[0377fac](https://github.com/keboola/keboola-as-code/pull/2087/commits/0377facfe1520062da1e095c3d68e991ce59c69e)